lib/srv/app/server.go

```diff
@@ -249,6 +255,10 @@ func New(ctx context.Context, c *Config) (*Server, error) {
 		return nil, trace.Wrap(err)
 	}
 
+	// Register handler that monitors tls connection state changes.
+	// This is used by HandleConnection to detect closed connections.
+	s.httpServer.ConnState = s.httpConnState.Notify
```

Comment:
Is this needed? Seems like this callback is already being set in `s.newHTTPServer()` call above.

Reply:
No. I moved it to newHTTPServer this morning and forgot to remove the original.